plugin.js

```diff
@@ -29,12 +30,13 @@ function plugin (fastify, options, next) {
   }
 
   fastify.decorateRequest('session', {})
-
+  const isNewSession = Symbol('isNewSession')
```

Comment:
I think this can be defined at the same level as `defaultOptions`.

Reply:
I tried this - the tests did not pass this way. I will try to figure out why

Reply:
Ok, I figured it out
Now the symbol is at the top like you suggested